storage/innobase/xtrabackup/src/xtrabackup.cc

```diff
@@ -2093,8 +2093,6 @@ dberr_t dict_load_tables_from_space_id(space_id_t space_id, THD *thd,
 			err = DB_ERROR;
 			goto error;
 		}
-
-		dd_table_close(ib_table, thd, nullptr, false);
```

Comment:
This looks inconsistent in terms of naming... `dd_table_open_on_dd_obj` has to be paired with `dd_table_close`